auctions/english-auction/src/events.sw

```diff
@@ -0,0 +1,28 @@
+library events;
+
+dep data_structures;
+
+use data_structures::{Asset, Auction};
+use std::identity::Identity;
+
+pub struct AuctionCancelEvent {
```

Comment:
```suggestion
pub struct AuctionCancelledEvent {
```

Reply:
I have all my events in the present tense. It doesn't make sense to me to have a past tense event to log something that is happening in that moment. The same applies to AuctionStartEvent below. I also wouldn't have a BiddedEvent.

Reply:
Yes that will work